src/xenia/base/threading_win.cc

```diff
@@ -315,8 +315,7 @@ class Win32Thread : public Win32Handle<Thread> {
   ~Win32Thread() = default;
 
   void set_name(std::string name) override {
-    AssertCallingThread();
-    xe::threading::set_name(name);
+    xe::threading::set_name(handle_, name);
```

Comment:
See https://github.com/benvanik/xenia/blob/master/src/xenia/base/threading_win.cc#L52.

In this case, you're passing it a handle where it needs a thread ID (which comes from `GetThreadId(handle_)`, and is different than the thread handle). I seem to remember `GetThreadId` behaving oddly (and always return ...FFFE, which is 'current thread id'), so it'd be nice to make sure that's not happening here.